pytorch_lightning/callbacks/lr_monitor.py

```diff
@@ -155,11 +157,23 @@ def _extract_stats(self, trainer, interval: str) -> Dict[str, float]:
 
         return latest_stat
 
-    def _extract_lr(self, param_group, name: str) -> Dict[str, float]:
+    def _extract_lr(self, trainer, param_group, name: str) -> Dict[str, float]:
         lr = param_group.get('lr')
         self.lrs[name].append(lr)
         return {name: lr}
 
+    def _remap_keys(self, names: List[str]) -> None:
+        token = '/pg1'
+        for new_name in names:
+            if token in new_name:
+                old_n = new_name.replace(token, '')
+                if old_n in self.lrs:
+                    self.lrs[new_name] = self.lrs[old_n]
+                    del self.lrs[old_n]
+            else:
+                if new_name not in self.lrs:
+                    self.lrs[new_name] = []
```

Comment:
Bit of refactoring, I believe it's the same

```suggestion
    def _remap_keys(self, names: List[str], token: str = '/pg1') -> None:
        for new in names:
            old = new.replace(token, '')
            if old in self.lrs:
                self.lrs[new] = self.lrs.pop(old)
            elif new == old:
                self.lrs[new] = []
```

Reply:
Done !

Reply:
Done !